src/main/scala/ai/datahunters/md/workflow/MainExtractionWorkflow.scala

```diff
@@ -25,7 +25,8 @@ class MainExtractionWorkflow(config: ProcessingConfig,
                              reader: PipelineSource,
                              writer: PipelineSink,
                              formatAdjustmentProcessor: Option[Processor] = None,
-                             analyticsFilters: Seq[Filter] = Seq()) extends Workflow {
+                             analyticsFilters: Seq[Filter] = Seq(),
+                             solrHashReader: Option[SolrHashReader] = None) extends Workflow {
```

Comment:
@dawidsau Before next release (not this one - coming) we should definitely change way of building Workflows. Too many arguments in constructor...